features/consent/src/main/res/layout/consent_details_fragment.xml

```diff
@@ -10,12 +10,20 @@
             type="in.projecteka.jataayu.core.model.Consent" />
 
         <variable
-            name="requestExpired"
-            type="Boolean" />
+            name="grantedConsentArtifact"
+            type="in.projecteka.jataayu.core.model.grantedconsent.GrantedConsentArtifactDetails" />
 
         <variable
             name="clickHandler"
             type="in.projecteka.jataayu.provider.ui.handler.ConsentDetailsClickHandler" />
+
+        <variable
+            name="requestExpiredOrGranted"
+            type="Boolean" />
+
+        <variable
+            name="isGrantedConsent"
```

Comment:
Since we already have this variable, do we really need `requestExpiredOrGranted`?

Reply:
We need  "isGrantedConsent" for showing granted consent specific data.